pkg/crosscluster/streamclient/partitioned_stream_client.go

```diff
@@ -406,6 +406,26 @@ func (p *partitionedStreamClient) CreateForTables(
 	return spec, nil
 }
 
+func (p *partitionedStreamClient) SetupReverseStream(
```

Comment:
I feel like `SetupReverseStream` is a bit of an odd man out compared to the other methods on the client. The other methods could be conceivably implemented over something like GRPC. `SetupReverseStream` on the other hand is accepting a sql command and is basically `conn.Exec` with a traced name.

I think this would match the pattern better if it took a struct that described the desired stream and internally constructed the statement. 

Alternatively, I think this should be named something that makes it clear what it really is: like `ExecStatement` or even expose the underlying pg connection object and run the statement on that.

Reply:
This is a good callout. I'll refactor this to be `ExecStatement`.

I prefer that over your first suggestion as it's much easier to construct the reverse stream statement during the planning of the original ldr stmt, where we access to the OG parsed statement. Then, since this needs to be called during job execution, i merely need to persist the cmd instead of all the components of command.